### ISSUE TITLE
Add titles to pages

### ISSUE THREAD

**Issue by paf31** [purescript/documentation#4]
It would be nice to add page titles if possible. I'm not sure if this is a quick fix by modifying the existing script or not, but we can close this if it's a hassle now.

**Comment by hdgarrood** [comment]
Yes, I agree. The reason I haven't done this yet is that I wasn't sure whether we intend people to read these docs on github or to use something like jekyll or sphinx (RTD) to generate HTML; the latter might add titles for us.
